controller/backing_image_controller.go

```diff
@@ -901,11 +901,36 @@ func (bic *BackingImageController) cleanupEvictionRequestedBackingImageCopies(bi
 		// only this controller can gather all the information of all the copies of this backing image at once.
 		// By deleting the disk from the spec, backing image manager controller will delete the copy on that disk.
 		// TODO: introduce a new CRD for the backing image copy so we can delete the copy like volume controller deletes replicas.
+		used, err := bic.isBIDiskFileUsedByReplicas(bi.Name, diskUUID)
+		if err != nil {
+			log.WithError(err).Warnf("Failed to check if the backing image copy on disk %v is used by replicas", diskUUID)
```

Comment:
If failed and is being used, how long will the controller continue retrying the deletion?

Reply:
The reconciliation will be triggered each time when replica get updated (deleted or finalized) or cleaned by K8s.

Reply:
Seems yes, WDYT? Any side effects?